lib/aggregate.js

```diff
@@ -29,12 +33,19 @@ module.exports = function(sourcePath) {
             var moduleMap = {};
 
             Object.keys(coverageData).forEach(function(moduleName) {
-                moduleMap[moduleName] = toPath(sourcePath, moduleName);
+                moduleMap[moduleName] = toPath(rootPath, moduleName);
+            });
+
+            var presentablePathMap = {};
+
+            Object.keys(coverageData).forEach(function(moduleName) {
+                presentablePathMap[moduleName] = revertOutOfRootFix(moduleName);
             });
 
             return {
                 coverageData: coverageData,
-                moduleMap: moduleMap
+                moduleMap: moduleMap,
+                presentablePathMap: presentablePathMap,
```

Comment:
And so we do the translation in a new dictionary and pass that on to the CLI table reporter and the Elm app.